python/lsst/afw/image/makeVisitInfo.py

```diff
@@ -73,25 +55,12 @@ def makeVisitInfo(
     @param[in] boresightRotAngle  rotation angle at boresight at middle of exposure;
                     see getBoresightRotAngle for details
                     (lsst.afw.geom.Angle, defaults to Angle(Nan))
-    @param[in] rotType  rotation type; one of the lsst.afw.image.RotType_ constants,
-                    defaults to RotType_UNKNOWN
+    @param[in] rotType  rotation type; one of the lsst.afw.image.RotType constants,
+                    defaults to RotType.UNKNOWN
     @param[in] observatory  observatory longitude, latitude and altitude,
                     (lsst.afw.coord.Observatory, defaults to Observatory(Angle(Nan), Angle(Nan), Nan))
     @param[in] weather  basic weather information for computing air mass,
                     (lsst.afw.coord.Weather, defaults to Weather(NaN, NaN, NaN))
     """
-    return VisitInfo(
-        exposureId,
-        exposureTime,
-        darkTime,
-        date,
-        ut1,
-        era,
-        boresightRaDec.toIcrs(),
-        boresightAzAlt,
-        boresightAirmass,
-        boresightRotAngle,
-        rotType,
-        observatory,
-        weather,
-    )
+    warnings.warn("makeVisitInfo is deprecated; use VisitInfo directly, instead", DeprecationWarning)
```

Comment:
This one was always documented as a temporary hack until we switched to pybind11. I can't speak to any others.